sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py

```diff
@@ -115,19 +115,19 @@ def register_schema(
         except AttributeError:
             pass
 
+        format = format.capitalize()
```

Comment:
If i'm understanding it correctly, `capitalization` is just for normalization of the returning SchemaProperties.format right?

Is capitalizing the input all languages agree upon?

Reply:
yeah it's just for normalization. I think so, since I put it in the sync group chat and Josh + Deya seemed to agree. I'll double check during the sync

Reply:
agreed on "Avro"